conbench/entities/benchmark_result.py

```diff
@@ -500,26 +499,29 @@ def ui_mean_and_uncertainty(self) -> str:
     @property
     def ui_rel_sem(self) -> Tuple[str, str]:
         """
-        The first string is a stringified float for sorting in the table.
-        The second string is for display in the table, with unit.
+        The first string in the tuple is a stringified float for sorting in a table.
+        The second string in the tuple is for display in the table, with unit.
+
         Associate "n/a" with a negative value so that DESC sorting by error
         is helpful.
         """
-        samples = self.data
+        if self.is_failed:
+            return (str(-1), "n/a (failed)")
 
-        if samples is None:
-            return (str(-1), "no data")
+        values = self.measurements
+        if all(v == 0 for v in values):
+            return (str(-1), "n/a (bad data)")
```

Comment:
Interesting case, isn't it? See https://github.com/conbench/conbench/issues/532.